{
  "config_missing_fixtests": [],
  "config_missing_tests": [],
  "config_with_errors": [],
  "fixtest_results": {},
  "results": {
    "rules/test_test/rule_that_timeout.yaml": {
      "checks": {
        "open-redirect": {
          "errors": [
            {
              "code": 2,
              "level": "warn",
              "message": "Timeout when running forcetimeout on targets/test_test/long.py:\n ",
              "path": "targets/test_test/long.py",
              "rule_id": "forcetimeout",
              "type": "Timeout"
            }
          ],
          "matches": "<masked in tests>",
          "passed": false
        }
      }
    }
  }
}